mqt/benchviewer/src/backend.py

```diff
@@ -150,14 +150,66 @@ def createDatabase(zip_file: ZipFile):
     return database
 
 
-def read_mqtbench_all_zip():
+def read_mqtbench_all_zip(
+    skip_question: bool = False,
+    target_location: str = "mqt/benchviewer/static/files/qasm_output",
+):
     global MQTBENCH_ALL_ZIP
-    huge_zip = Path("./static/files/qasm_output/MQTBench_all.zip")
+    huge_zip = Path(target_location + "/MQTBench_all.zip")
+    version_of_suitable_benchmark_zipfile = "v1.0.0"
```

Comment:
this seems to be very much hidden deep in the backend code and hardcoded. Shouldn't it be possible to determine the latest feasible tag in a programmatic/automatic way by querying GitHub?

Reply:
Removed fixed version, it is now deducted from pypi version number